swagger-spec/metaschemas/definition.yaml

```diff
@@ -0,0 +1,54 @@
+type: object
+title: Metaschema
+required:
+  - id
+  - type
+  - attributes
+  - links
+properties:
+  id:
+    type: string
+    readOnly: true
+    description: 'The unique identifier of the metaschema entity.'
+
+  type:
+    type: string
+    readOnly: true
+    description: 'The type identifier of the metaschema entity (`metaschemas`).'
+
+  attributes:
+    type: object
+    title: Attributes
+    readOnly: false
+    description: 'The properties of the metaschema entity.'
+    properties:
+      name:
+        type: string
+        readOnly: true
+        description: 'Name of the registration schema'
+      schema_version:
+        type: integer
+        readOnly: true
+        description: 'Latest version of the schema'
+      schema:
+        # TODO: Swagger UI seems to lack support for dictionaries.
+        # This property should not show as object
+        type: object
+        additionalProperties:
+          type: array
+          items:
+            type: string
+        readOnly: true
+        description: 'Registration schema details'
```

Comment:
"The details of the metaschema which contain the supplemental questions to accompany a registration."